src/rest/loggingHandler.go

```diff
@@ -10,7 +13,22 @@ type loggingHandler struct {
 }
 
 func (l *loggingHandler) handle(ctx iris.Context) {
-	logrus.Debugf("request: %s on %s", ctx.Request().Method, ctx.Request().URL.String())
+	// Enable reading the request body multiple times.
+	ctx.RecordRequestBody(true)
+
+	var reqInfo redfish.RequestInformation
+
+	// Retrieve request information from a request.
+	err := ctx.ReadJSON(&reqInfo)
+	if err != nil {
+		missingInfoError := "Unable to retrieve information from a request: " + err.Error()
+		logrus.Warn(missingInfoError)
+
+		ctx.Next()
+	}
+
+	requestUri := utils.UriTranslator.DmToRedfish(fmt.Sprintf("https://%s%s", reqInfo.Host, ctx.Request().RequestURI))
```

Comment:
There is no var UriTranslator any more, need to change to UriConverter.

Reply:
Fixed